mypy/semanal.py

```diff
@@ -1801,6 +1801,9 @@ def process_newtype_declaration(self, s: AssignmentStmt) -> None:
             self.fail(message.format(old_type), s)
             return
 
+        if 'unimported' in self.options.disallow_any and has_any_from_unimported_type(old_type):
```

Comment:
Seeing this finally made me realize why I'm not fond of the way you're organizing your flags. It's easy to imagine having a typo somewhere, e.g. `if 'unimprted in self.options.disalloy_any` and the mistake would only be detected by careful unit testing. OTOH if the test in the code was `if self.options.disallow_any_unimprted` the static checks would flag it right away. Note that this is independent from the way the command-line UI looks -- you could still have `--disallow-any=unimported` but it would have to set a Boolean attribute `disallow_any_unimported` in the Options object.